src/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+        content="width=device-width,
+          user-scalable=no,
+          initial-scale=1.0,
+          maximum-scale=1.0,
+          minimum-scale=1.0">
+    <title>Registration form</title>
+    <link rel="stylesheet" href="./css/style.css">
+  </head>
+  <body>
+
+    <form action="" name="registration" id="registrationForm">
+      <label for="fullName">Write your full name, please:</label>
+      <input type="text" name="fullname" id="fullName" placeholder="Full name" autocomplete="off">
+     
+      <div class="hidden-info">
+        <input type="checkbox" id="hidden-info" name="hidden-info" value="hidden-info">
```

Comment:
here you use hidden-info. Above - "registrationForm". 

use consistent naming.